grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -128,7 +132,7 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
      * */
     BigDecimal getInvoicedUnitPrice() {
         return invoiceItems?.sum {
-            if (it.invoice?.isRegularInvoice && !it.inverse) {
+            if (it.invoice?.datePosted != null && it.invoice?.isRegularInvoice && !it.inverse) {
```

Comment:
I don't like it here. This might break the `getUnitPriceAvailableToInvoice`. I think this should be a separate transient (invoiced unit price on posted invoices)

Reply:
(as you can see, on the `OrderItem` there are both `quantityInvoiced` and `postedQuantityInvoiced` for the same reason)